cmd/lotus-miner/sectors.go

```diff
@@ -314,17 +322,33 @@ var sectorsListCmd = &cli.Command{
 		var list []abi.SectorNumber
 
 		showRemoved := cctx.Bool("show-removed")
-		states := cctx.String("states")
-		if len(states) == 0 {
-			list, err = nodeApi.SectorsList(ctx)
-		} else {
+		var states []api.SectorState
+		if cctx.IsSet("states") && cctx.IsSet("unproven") {
+			return xerrors.Errorf("only one of --states or --unproven can be specified at once")
+		}
+
+		if cctx.IsSet("states") {
 			showRemoved = true
-			sList := strings.Split(states, ",")
-			ss := make([]api.SectorState, len(sList))
+			sList := strings.Split(cctx.String("states"), ",")
+			states = make([]api.SectorState, len(sList))
 			for i := range sList {
-				ss[i] = api.SectorState(sList[i])
+				states[i] = api.SectorState(sList[i])
+			}
+		}
+
+		if cctx.Bool("unproven") {
+			for state := range sealing.ExistSectorStateList {
+				if state == sealing.Proving {
```

Comment:
One minor point of confusion that I think is possible with this code and terminology is that there is a brief window where the sector is marked in onchain data as "unproven" but the sealing FSM sees the sector as "Proving".  If I understand the FSM correctly the sector transitions to FinalizeSector and then Proving after successful ProveCommit message without waiting for any window posts but miner state keeps the sector in "unproven" state until first post.

If there is another name you could give this flag (unfinalized, uncommitted, unporeped?) it would prevent confusion with the on chain concept.